File: src/Tawtheef.Websites/operations-web/src/app/pages/user/employee/locations-management/models/location.dto.ts
export interface LocationDto {
  id: string;
  nameAr: string;
  nameEn: string | null;
  locationLink: string;
  notes: string | null;
}

export interface LocationFilters {
  pageNumber: number;
  pageSize: number;
  search?: string;
}

export interface SaveLocationRequest {
  nameAr: string;
  nameEn: string | null;
  locationLink: string;
  notes: string | null;
}

File: src/Application.Operation/Features/Employee/Locations/DTOs/LocationDto.cs
namespace Application.Operation.Features.Employee.Locations.DTOs;

public sealed record LocationDto
{
    public Guid Id { get; init; }
    public required string NameAr { get; init; }
    public string? NameEn { get; init; }
    public required string LocationLink { get; init; }
    public string? Notes { get; init; }
}
